packages/from-hardhat/src/options.ts

```diff
@@ -0,0 +1,14 @@
+export interface EnvironmentOptions {
+  /**
+   * Working directory from which to interact with Hardhat
+   *
+   * @default process.cwd()
+   */
+  workingDirectory?: string;
+}
+
+export const withDefaultEnvironmentOptions = ({
+  workingDirectory = process.cwd()
+}: EnvironmentOptions = {}): EnvironmentOptions => ({
+  workingDirectory
+});
```

Comment:
I notice that you seem to prefer arrow function definitions. Why not use the `function withDefaultEnvironmentOptions(...) { ... }` syntax?

Personally I find plain old functions to be slightly more readable (less boilerplate) and they make it easier to inspect function references in a debugger, as you can often check their `name` property to find out where they originated from.

Curious to know what the benefits of using arrow functions like this might be. Perhaps it's that they break the global `this` binding?

Reply:
I literally only did it because I don't like `function() { return new Promise() }` as much as `() => new Promise()`

Reply:
I think this is fine as-is, I do prefer the plain old `function` syntax, but we do this in _many_ other places in the code, so it's not at all out of line.